### PR TITLE
move funcs in replicaset to autoscaling_utils

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
/kind cleanup

**What this PR does / why we need it**:

move funcs in replicaset to autoscaling_utils

**Which issue(s) this PR fixes**:

Ref: #86052

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/framework/replicaset/wait.go [hunk 1]
```diff
@@ -66,4 +55,4 @@ func WaitForReplicaSetTargetAvailableReplicas(c clientset.Interface, replicaSet
 		err = fmt.Errorf("replicaset %q never had desired number of .status.availableReplicas", replicaSet.Name)
 	}
 	return err
-}
+}
```
Comment (oomichi) [inline test/e2e/framework/replicaset/wait.go]: Need to fix here for solving the following verify job error:
```
diff -u ./test/e2e/framework/replicaset/wait.go.orig ./test/e2e/framework/replicaset/wait.go
--- ./test/e2e/framework/replicaset/wait.go.orig	2019-12-23 09:53:12.697540748 +0000
+++ ./test/e2e/framework/replicaset/wait.go	2019-12-23 09:53:12.697540748 +0000
@@ -55,4 +55,4 @@
 		err = fmt.Errorf("replicaset %q never had desired number of .status.availableReplicas", replicaSet.Name)
 	}
 	return err
-}
\ No newline at end of file
+}
```

Comment (oomichi) [reply test/e2e/framework/replicaset/wait.go]: /cc @oomichi

**Review by SataQiu — COMMENTED** [review]
Thanks
/approve
/test pull-kubernetes-e2e-gce

**Comment by oomichi** [comment]
Thanks for updating

/lgtm

**Pull request merged**